### ISSUE TITLE
Long training - learning rate decreases almost to zero after 30k steps for SmolVLA

### ISSUE THREAD

**Issue by Grigorij-Dudnik** [huggingface/lerobot#2558]
### System Info

```Shell
Default number of training steps for SmolVLA is 20k, but sometimes we want to train it longer, eg. in case of bigger datasets. Hovewer, learning rate curve decreases at same rate independently of nr of steps choosen, and model stops to learn after 30k steps.
```

### Information

- [ ] One of the scripts in the examples/ folder of LeRobot
- [x] My own task or dataset (give details below)

### Reproduction

I trained SmolVLA with parameter of steps set to 120000. Here is my learning rate curve:
<img width="795" height="444" alt="Image" src="https://github.com/user-attachments/assets/cb4ab16f-f187-4ae2-b80d-076f1aea693c" />

### Expected behavior

Can't say what expected behaviour should be. Maybe lr decrease rate should be proportional to nr of steps?

@jadechoghari

**Comment by jadechoghari** [comment]
@Grigorij-Dudnik can you share your training command?

**Comment by Grigorij-Dudnik** [comment]
Sure:

lerobot-train   --policy.path=lerobot/smolvla_base   --dataset.repo_id=Grigorij/right-arm-grab-notebook-sum  --batch_size=64   --steps=120000  --policy.repo_id=smolvla_right_arm_grab_notebook  --wandb.enable=true     --save_checkpoint true --save_freq 20000    --rename_map='{"observation.images.main": "observation.images.camera1", "observation.images.right_arm": "observation.images.camera2"}'

**Comment by yquanli** [comment]
Hi @Grigorij-Dudnik, this is not related to a bug but to hyperparameter settings. 
The default settings for the scheduler are as follows:
<img width="506" height="386" alt="Image" src="https://github.com/user-attachments/assets/c038e418-3da5-410b-b00e-80a6868fbf56" />
It looks like you've trained for more than 50k steps, but the scheduler decays the learning rate to 2.5e-6 after 30k steps.
You can change these parameters in your code, or use parameters like the one like this to assign values directly at the command line during training.
```
--policy.scheduler_warmup_steps=2000 \
--policy.scheduler_decay_steps=45000 \
```
After the modification, you will see that the learning rate is decaying in the way you expect, as shown in the comparison chart below:
<img width="782" height="530" alt="Image" src="https://github.com/user-attachments/assets/4efb5e57-b71e-47f0-88b8-9b985954560a" />

**Comment by Grigorij-Dudnik** [comment]
Oh, thank you for solution!

**Comment by imstevenpmwork** [comment]
Seems this has been clarified by: https://github.com/huggingface/lerobot/issues/2558#issuecomment-3649499921
